### ISSUE TITLE
BetterTTV not working

### ISSUE THREAD

**Issue by ASG1928** [night/betterttv#4934]
https://pixs.ru/image/GkSXva

**Comment by Unimatrix36** [comment]
Version 7.4.12 not working for me either on Chrome.
Can't access BTTV settings at all.

**Comment by Nareese** [comment]
This issue has been resolved in version 7.4.13, which has just been released, but will take some time before it is available on the various web browser store sites and automatically updates.  In the mean time, you can open a twitch page, open your browser's console window (Press F12 or right click and select Inspect), click on "Console" in the top menu bar, paste the following code into > or >> prompt at the bottom:
```
localStorage.bttv_settings = '{"anonChat":false,"autoTheatreMode":false,"splitChat":false,"deletedMessages":0,"reverseChatDirection":false,"pinnedHighlights":false,"timeoutHighlights":true,"highlightFeedback":false,"chatLayout":2,"tabCompletionTooltip":false,"tabCompletionEmotePriority":false,"whispers":true,"showDirectoryLiveTab":false,"channelPointsMessageHighlights":true,"clickTwitchEmotes":false,"darkenedMode":true,"friends":true,"primePromotions":true,"hostButton":false,"playerExtensions":true,"clickToPlay":false,"muteInvisiblePlayer":false,"scrollVolumeControl":false,"blacklistKeywords":{},"highlightKeywords":null,"sidebar":[15,0],"emotes":[7,0],"chat":[127,0],"autoPlay":[7,0],"usernames":[7,0],"channelPoints":[5,0],"version":"7.4.12"}'
```
That will reset all your settings to default (or create default settings if it's a fresh install), and should make 7.4.12 work again for you until you can get the update.
